src/main.h

```diff
@@ -1,6 +1,34 @@
 #ifndef NETDATA_MAIN_H
 #define NETDATA_MAIN_H 1
 
+/*! 
+ * \mainpage
+ *
+ * \tableofcontents
+ *
+ * \section sec_outline Outline
+ *
+ * \todo 
```

Comment:
This comment block starts with \mainpage.

This will be the initial start page of `doxygen`. I am not shure anymore if this is the right place to put it because your question mark. The todo is for me.

I think on the homepage of the `doxygen` documentation we should give an overview where to find wich part of net data. Also there should be additional information about coding rules etc. (Things the user of `netdata` does not need to no but the contributor)

What do you think?